kube/stateful_set.go

```diff
@@ -3,106 +3,70 @@ package kube
 import (
 	"fmt"
 
+	"github.com/SUSE/fissile/helm"
 	"github.com/SUSE/fissile/model"
-	"k8s.io/client-go/pkg/api/resource"
-	meta "k8s.io/client-go/pkg/api/unversioned"
-	"k8s.io/client-go/pkg/api/v1"
-	v1beta1 "k8s.io/client-go/pkg/apis/apps/v1beta1"
 )
 
-// NewStatefulSet returns a k8s stateful set for the given role
-func NewStatefulSet(role *model.Role, settings *ExportSettings) (*v1beta1.StatefulSet, *v1.List, error) {
+const volumeStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+
+// NewStatefulSet returns a stateful set for the given role
```

Comment:
Hmm, I guess the comment needs updating (to say that it returns the dependencies too).

Reply:
Updated in https://github.com/SUSE/fissile/commit/345a015c2d75aec4a02d8409ad458793f308aa83